pandas/core/algorithms.py

```diff
@@ -3,14 +3,18 @@
 intended for public consumption
 """
 from __future__ import division
+
 from warnings import warn, catch_warnings
-import numpy as np
 
 from pandas import compat, _np_version_under1p8
+from pandas.compat import string_types
+from pandas.compat.numpy import _np_version_under1p10
+from pandas.core import common as com
+
+import numpy as np
+from pandas._libs import algos, lib, hashtable as htable
+from pandas._libs.tslib import iNaT
 from pandas.core.dtypes.cast import maybe_promote
-from pandas.core.dtypes.generic import (
-    ABCSeries, ABCIndex,
```

Comment:
NOne, that was accidental, possibly due to Pydev or some other fluke, Ive tried to fix to baseline